### DISCUSSION TITLE
Unable to see Devices, after using confirmed LG/Apple ID Account

### DISCUSSION THREAD

**Discussion by PBXLthrMan** [nVuln/homebridge-lg-thinq#83] [General]
Hello,

Thank you for creating this plug-in first off. Unfortunately, I'm not sure if I am logging in correctly? I've selected LG Account, and put in my Apple ID and confirmed the password. And I confirmed I AM able to login into my LG ThinQ account, no issues.

But I am not seeing any devices listed?

I'm sure I am doing something wrong, LOL.

Thanks,

Cub

**Comment by nVuln** [comment]
if your account logged by Apple ID, please do not use password method, following here: [Wiki](https://github.com/nVuln/homebridge-lg-thinq/wiki/Plugin-Authorization-Setting)

**Reply by PBXLthrMan** [reply]
I’m not seeing that as option. 
![105792A0-3B19-4ACA-B8A1-F3AAD036E2CE](https://user-images.githubusercontent.com/90646855/133932624-e6577402-9128-4613-8578-1ec0834b0e7d.jpeg)

**Reply by nVuln** [reply]
You using Hoobs, please scroll down wiki page at [if you using Hoobs and logged by Google account or Apple ID or Facebook account](https://github.com/nVuln/homebridge-lg-thinq/wiki/Plugin-Authorization-Setting#if-you-using-hoobs-and-logged-by-google-account-or-apple-id-or-facebook-account), following that guide
you need account `refresh_token` to make plugin work or you can create new LG account (not logged by 3rd party) and migrate all device to this one, it's faster

**Reply by PBXLthrMan** [reply]
Got it. Thanks. I’ll try that next.

**Reply by Duracell79** [reply]
Hi can somebody help? How i can find device ID for homebridge LG ThinQ plugin settings? I'm trying to add LG TV

**Reply by nVuln** [reply]
@Duracell79 TV is not supported by this plugin, TV not use thinq platform, it's webos
